### ISSUE TITLE
Snowstorm pattern of uterus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10120]
**Preferred term label:**
Snowstorm pattern of uterus on ultrasound

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C2021434

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Molar pregnancy

**Your nano-attribution (ORCID)**
